src/utils/formatUserDetails.ts

```diff
@@ -22,6 +22,8 @@ export function formatUserDetails(userDetails: UserResponseType) {
   const userFullName = `**Full Name :** ${userDetails.user?.first_name} ${userDetails.user?.last_name}`;
   const discordJoinedAt = `**Joined Server on :** ${convertedTimestamp}`;
   const userState = `**State :** ${userDetails.user?.state}`;
+  const userName = `**UserName :** ${userDetails.user?.username}`;
+  const userId = `**UserId :** ${userDetails.user?.id}`;
 
-  return `## User Details\n${userFullName}\n${discordJoinedAt}\n${userState}`;
+  return `## User Details\n${userFullName}\n${discordJoinedAt}\n${userState}\n${userName}\n${userId}`;
```

Comment:
While printing the message, shouldn't we be consistent, and keep userName as **User Name** just like Full Name, and for userId to **User Id:**.

Because we are not creating this to store in DB, we are writing it as human readable english.

Reply:
Done